css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #0090e0;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+  background-color: #0090e0;
+  color: white;
+}
+
+.column-third {
+  height: 110px;
+  width: 110px;
+  margin: 10px 90px;
+  padding: -30px;
+  border-radius: 50%;
+  background-color: white;
+}
+
+img {
+  width: 200px;
+  padding: 70px;
+  margin-top: -42px;
+  margin-bottom: -40px;
+  margin-left: -43px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  display: flex;
+  flex-basis: auto;
+}
+
+h3 {
+  height: 150px;
+  width: 300px;
+  margin-left: -90px;
+  flex-basis: auto;
+  text-align: center;
+  justify-content: center;
+}
+
+p {
+  height: 150px;
+  width: 250px;
+  margin-top: -120px;
+  margin-left: -65px;
```

Comment:
Negative `padding` or `margin` is usually an indicator that you made an element too small. It is _highly_ unlikely to be the proper solution to a problem.